plaso/parsers/esedb_plugins/msie_webcache_cookies.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+"""Parser for the Microsoft Internet Explorer WebCache ESE database.
+
+The WebCache database (WebCacheV01.dat or WebCacheV24.dat) are used by MSIE
+as of version 10.
+"""
+
+from dfdatetime import filetime as dfdatetime_filetime
+from dfdatetime import semantic_time as dfdatetime_semantic_time
+
+from plaso.containers import events
+from plaso.lib import cookie_plugins_helper
+from plaso.parsers import esedb
+from plaso.parsers.esedb_plugins import interface
+
+class MsieWebCacheCookieData(events.EventData):
+  """MSIE WebCache Container table event data.
+
+  Attributes:
+    container_identifier (int): container identifier.
+    cookie_hash (str): a similarity hash of the cookie contents
+    cookie_name (str): name of the cookie
+    cookie_value_raw (str): raw value of cookie in hex
+    cookie_value (str): value of the cookie encoded in ascii
+    entry_identifier (int): entry identifier.
+    expiration_time (dfdatetime.DateTimeValues): expiration date and time.
+    flags (int): an representation of cookie flags
+    modification_time (dfdatetime.DateTimeValues): modification date and time.
+    rdomain (str): Request domain for which the cookie was set.
```

Comment:
style nit: don't use uncommon abbreviations, just make this `request_domain` or equivalent.